examples/qualcomm/oss_scripts/ssd300_vgg16.py

```diff
@@ -109,7 +109,9 @@ def SSD300VGG16(pretrained_weight_model):
     from model import SSD300
 
     model = SSD300(n_classes=21)
-    checkpoint = torch.load(pretrained_weight_model, map_location="cpu")
+    checkpoint = torch.load(
+        pretrained_weight_model, map_location="cpu", weights_only=False
```

Comment:
Wouldn't `weights_only=True` work here and in other places?

`weights_only=False` is a potential security issue.
I guess it's fine to don't cause any compatibility problems right now, but maybe add a TODO to investigate later?

Reply:
Let me add them as TODO then, these are examples from different partners, so I guess setting `weights_only` to True would better be reviewed separately.